tasks/release/bump_version.py

```diff
@@ -0,0 +1,75 @@
+"""Helper Commands for making releases
+"""
+
+import os
+
+from .utils import ROOT, build_version, find_version, replace_in_file, Version
+
+from invoke import task
+
+
+def _bump_version_str(version_str, major=False, minor=False, patch=False,
```

Comment:
yet another version bumping system?

Reply:
It's super trivial?

```
invoke release.bump-version --minor/--major/--patch/--dev/--final
```

Anyway, it won't be too hard to switch to `setuptools_scm` or something if it's what others want to do.